src/dialog.jsx

```diff
@@ -337,7 +337,7 @@ let Dialog = React.createClass({
       let contentHeight = dialogWindowHeight - dialogContentPadding;
 
       if (this.props.title) contentHeight -= dialogContent.previousSibling.offsetHeight;
-      if (this.props.actions) contentHeight -= dialogContent.nextSibling.offsetHeight;
```

Comment:
@hackingbeauty I think you may have to do:
```
if (this.props.actions && this.props.actions.length) ...
```

Also, it looks like this PR conflicts with some recent commits. Can you rebase please?